pjsip-apps/src/pjsua/pjsua_app_legacy.c

```diff
@@ -248,6 +248,8 @@ static void keystroke_help()
     puts("| dq  Dump curr. call quality  | cd  Disconnect port      | dc  Dump config   |");
     puts("|                              |  V  Adjust audio Volume  |  f  Save config   |");
     puts("|  S  Send arbitrary REQUEST   | Cp  Codec priorities     |                   |");
+    puts("| +l Set Late Offer Answer Mode   |                       |                   |");
+    puts("| -l Unset Late Offer Answer Mode |                       |                   |");
```

Comment:
Cosmetics:
- alignment, need one more space char before "Set" & "Unset" to be aligned to "Send" above it.
- can the words be a bit shortened, e.g: "Set/Unset late offer" (the term used in the provided link seems to be "Late Offer" instead of "Late Offer Answer Mode"?)

Reply:
Perhaps better to change it to: "Toggle call SDP offer"?
So it would  look like this:
```
|  a  Answer call              |  i  Send IM              | !a  Modify accnt. |
|  h  Hangup call  (ha=all)    |  s  Subscribe presence   | rr  (Re-)register |
|  H  Hold call                |  u  Unsubscribe presence | ru  Unregister    |
|  o  Toggle call SDP offer    |  D  Subscribe dlg event  |
```

Also, after the command is executed, need a message for the user to understand that the flag `PJSUA_CALL_NO_SDP_OFFER` only applies for next call and next UPDATE, but not re-INVITE (i.e. hold and uphold), something like this:
`Subsequent calls and UPDATEs will contain SDP offer: [yes/no]`

Reply:
Guys, please, support me with the implementation of your comments.
I didn't understand how to implement LOAM in the CLI menu.
I really appreciate any help you can provide.

Reply:
Please find the patch. Feel free to modify it
[patch_cli.patch](https://github.com/pjsip/pjproject/files/14566880/patch_cli.patch)